orc8r/cloud/deploy/terraform/orc8r-helm-aws/logging.tf

```diff
@@ -18,7 +18,7 @@ locals {
 resource "helm_release" "fluentd" {
   count = var.elasticsearch_endpoint == null ? 0 : 1
 
-  name       = "fluentd"
+  name       = var.external_dns_deployment_name
```

Comment:
This should be `var.fluentd_deployment_name` right?

Reply:
yes. 100%